generator/maxspeeds_builder.cpp

```diff
@@ -246,7 +246,10 @@ class MaxspeedsMwmCollector
             break;
         }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
         if (status == 0)
+#pragma GCC diagnostic pop
```

Comment:
Eww... How about just initializing it? :)

line 181: 
```cpp
int status = 0;
```

Reply:
Previosly vng and biodranik were against extra initializations, just to avoid compiler bugs. Hence the manual control of the compiler.

Reply:
That sounds weird. I don't like this approach
@vng could you elaborate please?

Reply:
- This kind of warning suppression is ugly and makes code less readable.
- ```int status = 0;``` here is fine

Reply:
Changed.